src/andromede/libs/antares_historic/antares_historic.yml

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+library:
+  id: antares-historic
+  description: Antares historic model library
+
+  port-types:
+    - id: flow
+      description: A port which transfers power flow
+      fields:
+        - name: flow
+
+  models:
+    - id: area
+      parameters:
+        - name: spillage_cost
+        - name: ens_cost
+      variables:
+        - name: spillage
+          lower-bound: 0
+          type: float
+        - name: unsupplied_energy
+          lower-bound: 0
+          type: float
+      ports:
+        - name: balance_port
+          type: flow
+      binding-constraints:
+        - name: balance
+          expression: sum_connections(balance_port.flow) = spillage - unsupplied_energy
+      objective: expec(sum(spillage_cost * spillage + ens_cost * unsupplied_energy))
+
+    - id: load
+      parameters:
+        - name: load
+      ports:
+        - name: balance_port
+          type: flow
+      port-field-definitions:
+        - port: balance_port
+          field: flow
+          definition: -load
+
+    - id: link
+      parameters:
+        - name: capacity
+          time-dependent: false
+          scenario-dependent: false
+      variables:
+        - name: input
+          lower-bound: -capacity
+          upper-bound: capacity
+      ports:
+        - name: out_port
+          type: flow
+        - name: in_port
+          type: flow
+      port-field-definitions:
+        - port: out_port
+          field: flow
+          definition: input
+        - port: in_port
+          field: flow
+          definition: -input
+
+    - id: renewable
+      parameters:
+        - name: nominal_capacity
+          time-dependent: false
+          scenario-dependent: false
+        - name: unit_count
+          time-dependent: false
+          scenario-dependent: false
+        - name: generation
+          time-dependent: true
+          scenario-dependent: true
+      ports:
+        - name: balance_port
+          type: flow
+      port-field-definitions:
+        - port: balance_port
+          field: flow
+          definition: generation
+
+    - id: thermal_cluster
+      parameters:
+        - name: p_min_cluster
+        - name: p_max_cluster
+        - name: p_min_unit
+        - name: p_max_unit
+        - name: generation_cost
+        - name: startup_cost
+        - name: fixed_cost
+        - name: d_min_up
+          type: integer
+        - name: d_min_down
+          type: integer
+        - name: nb_units_min
```

Comment:
Parameter deduced from p_min_cluster

Reply:
Is it simply p_min_cluster / p_min_units ?

Reply:
p_min_cluster / p_max_units (minimum number of units needed to produce p_min_cluster)

Reply:
Same as for `nb_units_max` cf https://github.com/AntaresSimulatorTeam/andromede-modeling-prototype/pull/57#discussion_r1894214498